packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -279,44 +291,6 @@ export default function AccessRequirementList(
     }
   }, [numberOfFilesAffected, requestObjectName, subjectId, subjectType])
 
-  const content = (
```

Comment:
This content is only shown when the current step is `SHOW_ALL_ARS`, so this was moved into the `switch` where all of the other content is created